### PR TITLE
feat: [GM-19] [BE] Design the get user endpoint proto

### PR DESCRIPTION
**Pull request by krishankumar01**

### CODE REVIEW

**Review by hwbrzzl — DISMISSED** [review]
Great!

**Comment by hwbrzzl** [comment]
CI failed, it's not related to your change, but could you please help fix it in this PR? I guess it should not be complex.

**Comment by krishankumar01** [comment]
> CI failed, it's not related to your change, but could you please help fix it in this PR? I guess it should not be complex.

fixed

**Review by hwbrzzl — APPROVED** [review]

File: src/go/user/lang/cn.json [hunk 1]
```diff
@@ -22,5 +22,8 @@
   },
   "exist": {
     "email": "邮箱已存在"
+  },
+  "not_exist": {
+      "user": "找不到用户"
```
Comment (hwbrzzl) [inline src/go/user/lang/cn.json:27]: You learned a new sentence. 👍

**Pull request merged**